.travis.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     name: Lint
   - script: npm run lint
     name: Tests
-    script: npm run test && npx codecov
+    script: npm run test:ct && npx codecov
```

Comment:
We should combine both reports from jest and cypress If I understand it right, by this change, you are not running jest and cypress tests together in the same job. This will lead to a situation, where only `cypress-coverage` is generated. Thus, `npx codecov` will report only the coverage from cypress tests suite.

Reply:
Yeah, you are correct, need to update the script